src/services/ClaudyActionComponents/Support.jsx

```diff
@@ -77,12 +82,14 @@ export class Support extends Component {
                 onChange={(e) => { this.setState({message: e.target.value}) }}
               />
             </label>
-            {!isSent && !isSending && !error &&
+            {(
+              (!isSent && !isSending && !error) ||
+              (isSent && !isSending && !error && message)) &&
```

Comment:
Shouldn't we be using some kind of selectors here ? The code readibility is not optimal imho.